### ISSUE TITLE
SSD MobileNet v1 bounding boxes all over the place

### ISSUE THREAD

**Issue by Charios** [tensorflow/models#4128]
## **System information**
- **What is the top-level directory of the model you are using**
object_detection
- **Have I written custom code**
No
- **OS Platform and Distribution**
Windows 10
- **TensorFlow installed from**
Source
- **TensorFlow version**
1.5
- **Bazel version**
- **CUDA/cuDNN version**
CUDA 9.0
cuDNN 7.0
- **GPU model and memory**
GTX 970 4GB

I've trained SSD MobileNet v1 on my own dataset of ~4k dog pictures and it displays bounding boxes all over the place. I've trained with batch size 1.
![31444603_1934636123235225_6525328454009749504_n](https://user-images.githubusercontent.com/9070833/39424518-c8bbbbe6-4c7f-11e8-8c84-31421d049284.png)
The same dataset trained on faster rcnn works really well, and detects dogs properly.
I've run 50k iterations, and total loss was spiking from 1.5 to 3 all the time in the end, with random bigger spikes, but overall it went down from 20

There wasn't a smooth and gradual decline, but a really spiky and uneven one with both models

Learning graph
![8muea7g](https://user-images.githubusercontent.com/9070833/39436380-7e2d27aa-4ca6-11e8-97e9-8dcfda43c65b.png)

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by robieta** [comment]
This is more of a [TensorFlow Stack Overflow](http://stackoverflow.com/questions/tagged/tensorflow) question, as it really will just come down to hyperparameter tuning.

@pkulzc Do you want to take a look or shall I close it.

**Comment by pkulzc** [comment]
I agree with @robieta. Should be OK to close this.
